core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4137,6 +4153,8 @@ private Name() {} // prevent instantiation
   @ThreadSafe
   public enum Template {
     LOCALITY_TIER("alluxio.locality.%s", "alluxio\\.locality\\.(\\w+)"),
+    PATH_INDEX("alluxio.path.%d", "alluxio\\.path\\.(\\d+)"),
```

Comment:
my intention is to add a new configuration file: alluxio-path.properties, CLI persists path level configs into this file, AlluxioProperties load this file, the properties PATHS, PATH_INDEX, and PATH_PROPERTY can only be set in and loaded from this file.

Reply:
we used to have multiple properties files a few years ago, then decided it was too confusing, and merged them all into alluxio-site.properties. I don't think we want to go back to multiple files.

path level defaults most closely mirror mount properties, so I'd like to manage them in a similar way, i.e. 100% from the command line, with the journal as the source of truth. We could add a feature to update many defaults at once via file, but I don't see that as a core part of this work.

since path level defaults are managed differently from property keys, I don't think they should be included in `PropertyKey.java`. Then we don't have to worry about users setting them in `alluxio-site.properties`